kosmorrolib/main.py

```diff
@@ -99,14 +100,30 @@ def main():
     return 0
 
 
-def get_date(yyyymmdd: str) -> date:
-    if not re.match(r'^\d{4}-\d{2}-\d{2}$', yyyymmdd):
-        raise ValueError(_('The date {date} does not match the required YYYY-MM-DD format.').format(date=yyyymmdd))
+def get_date(date_arg: str) -> date:
```

Comment:
Maybe we should move this function to `core.py` and add some unit tests on it… WDYT?

Reply:
Yeah that would be nice, but the functions output currently changes with the current date. So we would have to add something like `current_date` to the parameters of the function.

Reply:
Actually, you can use the same `relativedelta()` function in your test to check you get the same date as expected, no matter what the current date is. The goal is just to check the string is interpreted as expected :)

Reply:
Ah ok